godotdocset.py

```diff
@@ -131,23 +131,39 @@ def get_plist(name: str):
 	<string>dashtoc</string>
 </dict>
 </plist>
-'''.format(name.split('_')[0],
-		name.replace('_', ' '),
-		name.split('_')[0].lower())
+'''.format(name.split('_')[0]+version, name.replace('_', ' ')+version, name.split('_')[0].lower())
 
 
+def get_args_from():
+	ap = argparse.ArgumentParser()
+	ap.add_argument('-f', '--from', help="folder or xml file", required=True)
+	# ap.add_argument('-t', '--to', help="output folder", default='.')
+	args = ap.parse_args()
+	frompath = args.__dict__['from']
+	return frompath
+
+def get_version():
+	VERSION_PATH = os.path.abspath(os.path.join(get_args_from(), "../../version.py"))
+	version = ""
+	for line in open(VERSION_PATH):
+		if line.startswith("docs ="):
+			version = "-"+line.split("=")[1].strip().strip('\"')
+	return version
+
 class DocsetMaker:
 	outname = "Godot"
-	rootdir = outname + '.docset'
+	v_outname = outname+get_version()
+	# version_path = os.path.abspath(os.path.join("../", frompath, "./version.py"))
+	rootdir = outname + get_version() + '.docset'
```

Comment:
`v_outname + ".docset"`?

Reply:
All fixed with new commit, sorry I'm not a python dev, but all these comments are useful to learn